### PR TITLE
Chapter1/homework/3

### PR DESCRIPTION
**Pull request by jasonlhoward**
Completed homework 3. I liked the names of the functions in Checks.cs so I used those names as well. Thanks.

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275&bid=16252895)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        {
+            Console.Write(message);
+            return Console.ReadLine();
+        }
+
+        public static float PromptFloat(string message)
+        {
+            Console.Write(message);
+            string input = Console.ReadLine();
+            // TODO: check input can be parsed as float
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275)

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -9,6 +9,27 @@ class Program
...
+            // 2) Take homework 2 code, place it in the forked branch and refactor it using functions.There should be as little duplicate code as possible (there should be functions for:
+            // -Calculating BMI(weight comes in kg, height comes in meters),
+            // -Prompt for input and converting it to int(print message for request, read console input and return converted input to int), 
+            // -Prompt for input and converting it to string(print message for request, read console input and return input),
+            // -Prompt for input and converting it to float(print message for request, read console input and return converted input to float).
+            // 3) Put all the function you made into the right places in Checks.cs class. Run tests, make sure you pass all the tests
+            // 4) Put all program.cs logic to Lesson3.cs.Call it from main function. Program class should look like this:
+
+            //public class Program
+            //        {
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+            float bmi = CalculateBmi(weight, height);
+
+            Console.WriteLine("{0} {1} is {2} years old, he has a BMI of {3} with a weight of {4} kg and a height of {5} meters.", firstName, lastName, age, bmi, weight, height);
+         }
+
+        public static int PromptInt(string message)
+        {
+            Console.Write(message);
+            string input = Console.ReadLine();
+            // TODO: check input can be parsed as int
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
- Added 3
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275&bid=16252895)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        {
+            Console.Write(message);
+            return Console.ReadLine();
+        }
+
+        public static float PromptFloat(string message)
+        {
+            Console.Write(message);
+            string input = Console.ReadLine();
+            // TODO: check input can be parsed as float
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275)

File: Src/BootCamp.Chapter/Program.cs [hunk 7]
```diff
@@ -1,14 +1,30 @@
...
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+            float bmi = CalculateBmi(weight, height);
+
+            Console.WriteLine("{0} {1} is {2} years old, he has a BMI of {3} with a weight of {4} kg and a height of {5} meters.", firstName, lastName, age, bmi, weight, height);
+         }
+
+        public static int PromptInt(string message)
+        {
+            Console.Write(message);
+            string input = Console.ReadLine();
+            // TODO: check input can be parsed as int
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
- Added 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275&bid=16252895)

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 8 above)
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275)

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 6 above)
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275)

**Comment by jasonlhoward** [comment]
Awesome. Thanks.

**Pull request closed without merging**